dp3t-sdk/sdk/src/main/java/org/dpppt/android/sdk/util/DateUtil.java

```diff
@@ -25,4 +25,8 @@ public static int getRollingStartNumberForDate(DayDate onsetDate) {
 		return getRollingStartNumberForDate(onsetDate.getStartOfDayTimestamp());
 	}
 
+	public static DayDate getDayDateForRollingStartNumber(int rollingStartNumber) {
+		return new DayDate(rollingStartNumber / 24 / 6 * 1000L * 60 * 60 * 24);
```

Comment:
this calculation is too complicated: new DayDate(rollingStartNumber*1000L*60*10) will give the same result in my opinion and is less confusing, DayDate will automatically round to the UTC day.